content/en/tracing/trace_collection/otel_instrumentation/java.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Custom Instrumentation of Java Applications with the OpenTelemetry API
+kind: documentation
+description: 'Instrument your Java application with OTel API to send traces to Datadog.'
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 0
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
```

Comment:
Just noting that we have further reading links in the front matter, but we don't have the Further Reading section at the bottom of the article.